frontend/uddhar-frontend/src/coordinator/components/OngoingEvent.jsx

```
@@ -0,0 +1,23 @@
+import { MapPin } from "lucide-react";
+
+const OngoingEvent = ({ info, onClickEventHandler, bg}) => {
+  return (
+    <>
+      <div
+        className={`min-w-70 h-50 ${bg} flex flex-col gap-y-[10px] text-left p-10 rounded cursor-pointer`}
+        onClick={() => {
+          onClickEventHandler(info.location);
+        }}
+      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)